vortex-file/src/read/layouts/chunked.rs

```diff
@@ -77,13 +81,38 @@ impl ChunkedLayout {
         }
     }
 
+    fn metadata_layout(&self) -> VortexResult<Option<Box<dyn LayoutReader>>> {
```

Comment:
I need very similar logic to expose statistics to datafusion, WDYT about splitting it out to a different PR we can quickly merge?

Reply:
How does this function know where in the layout tree it is?

Reply:
oh, this method is local, eh ignore me

Reply:
@AdamGS Yeah I can make PR

Reply:
https://github.com/spiraldb/vortex/pull/1399